packages/augur-artifacts/src/environments/kovan.json

```
@@ -32,7 +32,7 @@
       "ws": "wss://v2.augur.net/0x-ws"
     },
     "mesh": {
-      "verbosity": 1,
+      "verbosity": 5,
```

```suggestion
      "verbosity": 1,
```

Do we want it this noisy by default?

---

ugh I changed that for testing. one sec...